wagtailio/blog/blocks.py

```diff
@@ -0,0 +1,6 @@
+from wagtailio.core.blocks import StandaloneCTABlock
+from wagtailio.utils.blocks import StoryBlock
+
+
+class BlogStoryBlock(StoryBlock):
+    standalone_cta = StandaloneCTABlock()
```

Comment:
question: `StoryBlock` still uses the old streamfield template (https://github.com/wagtail/wagtail.org/blob/127f2be58f11fbac39907d0d3aa80181e62afcf9/wagtailio/utils/blocks.py#L197), and some of its blocks, like the backer blocks reference templates from the blog app ([example](https://github.com/wagtail/wagtail.org/blob/127f2be58f11fbac39907d0d3aa80181e62afcf9/wagtailio/utils/blocks.py#L159)). Will the FE ticket cover this? if not, can you update the relevant paths?

Reply:
Need to counter-check with @chris-lawton / @William-Blackie on this one

Reply:
this is the last bit. everything else LGTM as is.
Note: we may want to revisit the blog post streamfield blocks in the future, but that will do for now

Reply:
Hey @engineervix, 

I'll reply in here instead of on Slack, I would just change it here to the correct template, even if it's done elsewhere in another MR there's no clash and will help with a review (it can't be reviewed if we can't see it!). 

Thanks - Will.

Reply:
Thanks @zerolab & @William-Blackie. I have updated the template references via 230312a. I've also added the block templates in `wagtailio/utils/blocks.py` to the Pattern Library via ba9ab50

Reply:
Hey @engineervix, great work! As we're migrating over to a new layout for templates (which you've gone ahead and done), the logical next step would be to remove the old ones we no longer reference in this directory [here](https://github.com/wagtail/wagtail.org/blob/6f9e6e556e24f1ff13d70157802212faca7c36a7/wagtailio/blog/templates/blog/blocks/image_backer.html).

We'll also need to style these blocks as it looks like it's been removed (most likely when Chris and I initially tooled the project up). Just raise this in standup and we can deal with it.

Thanks!

Reply:
Thanks @William-Blackie 

Done in 423bd9b